.github/workflows/updater-check.yml

```diff
@@ -0,0 +1,39 @@
+---
+on:
+  push:
```

Comment:
I am guessing I need to remove this `push` while merging to the master? @gavin-stackrox